Cargo.toml

```diff
@@ -4,13 +4,20 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-clap = { version = "4.5.21", features = ["derive"] }
 anyhow = { version = "1.0.93" }
-tokio = { version = "1.41.1" }
-aptos-sdk = { git = "https://github.com/aptos-labs/aptos-core", rev = "28abb7db64af8de81a6bc76c6b2d6600181ac578" }
-serde = { version = "1.0.215" }
+aptos = { git = "https://github.com/sota-zk-labs/aptos-core", branch = "jayce-aptos-node-v1.23.5" }
+aptos-sdk = { git = "https://github.com/sota-zk-labs/aptos-core", branch = "jayce-aptos-node-v1.23.5" }
+clap = { version = "4.5.21", features = ["derive"] }
 config = { version = "0.14.1" }
+dialoguer = "0.11.0"
+rand = "0.7.3"
```

Comment:
version =

Reply:
the latest version of `rand` is not compatiable